app/models/edition.rb

```diff
@@ -247,15 +248,6 @@ def self.without_locked_documents
     .where.not("documents.locked = true")
   end
 
-  def self.latest_published_edition
```

Comment:
just in case scopes get misused in future i wonder if this could've just been 

```
def self.latest_published_edition
   joins(:document).where("documents.live_edition_id = editions.id AND editions.state = 'published'") }
end 
```

not big deal either way though!